docs/installation/ha-installation-with-cli.md

```diff
@@ -0,0 +1,69 @@
+---
+title: High Availability Installation Using CLI
+---
+
+This documentation serves as a comprehensive guide for deploying Karmada in a high
+availability (HA) configuration using the `karmadactl` CLI tool.
+
+## Prerequisites
+
+- A Kubernetes cluster with multiple worker nodes (it's recommended to have at least
+  three worker nodes).
+- A valid `kube-config` file to access your Kubernetes cluster.
+- The `karmadactl` command-line tool or `kubectl-karmada` plugin installed.
+
+  :::note
+
+  For installing `karmadactl` command line tool refer to the [Installation of CLI Tools](/docs/next/installation/install-cli-tools#one-click-installation) guide.
+
+  :::
+
+## Installation
+
+Follow these steps to install Karmada with high availability:
+
+1. Use the following command to install the Karmada in HA:
+
+  ```bash
+  sudo karmadactl init --karmada-apiserver-replicas 3 --etcd-replicas 3 --etcd-storage-mode PVC --storage-classes-name <storage-classes-name> --kubeconfig <path-to-kube-config>
+  ```
+
+  :::note
+
+  You need to use sudo for elevated permissions because `karmadactl` creates a
+  `karmada-config` file in the `/etc/karmada/karmada-apiserver.config` directory.
+
+  :::
+
+2. Specify the path to your `kube-config` file to connect to your Kubernetes cluster.
+  Typically, the `kube-config` file is located at `~/.kube/config` directory.
+  Alternatively, you can set the `KUBECONFIG` environment variable to specify the
+  file's location.
+
+3. Adjust the installation parameters as needed:
+   - `--karmada-apiserver-replicas 3`: This sets up three Karmada API server replicas.
+   - `--etcd-replicas 3`: This ensures three etcd members are available to support the
+      three Karmada API servers.
+   - `--etcd-storage-mode PVC`: It indicates the use of PVCs for etcd storage.
+   - `--storage-classes-name <StorageClassesName>`: Specify the name of the storage
+      class for etcd.
+
+  :::note
+
+  Make sure you have a minimum of three Kubernetes worker nodes available for a
+  successful installation of three Karmada API servers and etcd replicas.
+  Otherwise, the installation will not work.
+
+  :::
```

Comment:
```suggestion
   :::note

   Make sure you have a minimum of three Kubernetes worker nodes available for a
   successful installation of three Karmada API servers and etcd replicas.
   Otherwise, the installation will not work.

   :::
```